admin/test/features/support/sql_db_helper.rb

```diff
@@ -0,0 +1,44 @@
+class SqlDbHelper
+
+  def self.pupil_details(upn)
+    sql = "SELECT * FROM [mtc_admin].[pupil] WHERE upn='#{upn}'"
+    result = SQL_CLIENT.execute(sql)
+    result.first
+  end
+
+  def self.pupil_details_using_names(firstname, lastname)
+    sql = "SELECT * FROM [mtc_admin].[pupil] WHERE foreName='#{firstname}' AND lastName='#{lastname}'"
+    result = SQL_CLIENT.execute(sql)
+    result.first
+  end
+
+  def self.find_teacher(name)
+    @array_of_users = []
+    sql = "SELECT * FROM [mtc_admin].[user] WHERE identifier='#{name}'"
```

Comment:
identifier is unique, so you will only ever get one result

Reply:
Sometimes this is used to dereference the container array.

Reply:
👍